### PR TITLE
Updated build number for 4.3.1

### PR DESCRIPTION
**Pull request by jainaashish**
Updated build number for 4.3.1

@rrelyea

### CODE REVIEW

**Comment by rohit21agrawal** [comment]
this is the right way to do it: https://github.com/NuGet/NuGet.Client/pull/1577

**Review by emgarten — APPROVED** [review]

**Review by rohit21agrawal — CHANGES_REQUESTED** [review]
follow changes in : https://github.com/NuGet/NuGet.Client/pull/1577

**Comment by rohit21agrawal** [comment]
this is more apt actually : https://github.com/NuGet/NuGet.Client/commit/20f3ac502638ac03d8354f5e2ba3d8186fa77645

**Comment by jainaashish** [comment]
@rohit21agrawal let me know if there are more commits as well with different changes... I'll take care of all these in one shot!

**Comment by rohit21agrawal** [comment]
lol..sorry about that..this is all as far as i know..you do need to create a new branch in NuGet.Build.Localization as well

File: .gitmodules [hunk 1]
```diff
@@ -13,4 +13,4 @@
 [submodule "submodules/NuGet.Build.Localization"]
 	path = submodules/NuGet.Build.Localization
 	url = https://github.com/NuGet/NuGet.Build.Localization.git
-	branch = release-4.3.0-rtm
+	branch = release-4.3.1-rtm
```
Comment (rohit21agrawal) [inline .gitmodules]: i thought the branch name was release-4.3.1 ?

File: scripts/cibuild/ConfigureVstsBuild.ps1 [hunk 2]
```diff
@@ -122,7 +122,7 @@ $NuGetClientRoot = $env:BUILD_REPOSITORY_LOCALPATH
 $Submodules = Join-Path $NuGetClientRoot submodules -Resolve
 
 $NuGetLocalization = Join-Path $Submodules NuGet.Build.Localization -Resolve
-$NuGetLocalizationRepoBranch = 'release-4.3.0-rtm'
+$NuGetLocalizationRepoBranch = 'release-4.3.1-rtm'
```
Comment (rohit21agrawal) [inline scripts/cibuild/ConfigureVstsBuild.ps1]: same here

File: .gitmodules (hunk 1 above)
Comment (jainaashish) [reply .gitmodules]: oh yes, i thought it's just the release label

File: scripts/cibuild/ConfigureVstsBuild.ps1 (hunk 2 above)
Comment (jainaashish) [reply scripts/cibuild/ConfigureVstsBuild.ps1]: do we need to update anything in NuGet.Build.Localization branch as well?

Comment (rohit21agrawal) [reply scripts/cibuild/ConfigureVstsBuild.ps1]: no, just create a new branch with the name that's specified here.

Comment (jainaashish) [reply scripts/cibuild/ConfigureVstsBuild.ps1]: ok, then i'll keep it consistent

**Review by rohit21agrawal — APPROVED** [review]

**Pull request merged**